tests/mixed_int8/README.md

```diff
@@ -1,37 +1,122 @@
 # Testing mixed int8 quantization
 
+![HFxbitsandbytes.png](https://s3.amazonaws.com/moonup/production/uploads/1660567705337-62441d1d9fdefb55a0b7d12c.png)
+
+Hi there, this is a recipe on how to effectively debug `bitsandbytes` integration on HuggingFace `transformers`.
+
+## Library requirements
+
++ `transformers>=4.22.0`
++ `accelerate>=0.12.0` 
++ `0.31.8>=bitsandbytes>=0.31.5`.
```

Comment:
Is `0.31.8` necessary? Or it is just the version that has been tested without problem?

Reply:
Actually we know that these versions work and the latest version is `0.31.8` , so yes it's the versions we know it has been tested without problem

Reply:
Actually there is no reason, proposed an update below
```suggestion
+ `bitsandbytes>=0.31.5`.
```